### ISSUE TITLE
Reading / Writing Pivot Table

### ISSUE THREAD

**Issue by cbeuthner** [PHPOffice/PhpSpreadsheet#1017]
This is:

```
- [x ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Load existing template with a pivot table in a worksheet, input data in a second sheet, and have the pivot table updated with the new data.

### What is the current behavior?
The pivot table isn't working after saving the file.

### What are the steps to reproduce?
Just use a Excel-file with a working pivot table. Open it, add some data and write/save the file.

### Which versions of PhpSpreadsheet and PHP are affected?
PHP 5.6.45
PHPSpreadsheet 1.7

**Comment by Collie-IT** [comment]
There is current no Pivot support.

**Comment by oleibman** [comment]
Just implemented with PR #4942. Leaving issue closed, removing stale label.
